packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -272,7 +272,7 @@ export const MenuItem = forwardRef<HTMLLIElement, MenuItemProps>(
           <Text
             as="div"
             className={`${prefix}--menu-item__label`}
-            title={label}>
+            title={rest?.title || label}>
```

Comment:
I don't think it's a good idea to encourage the `title` be something different than the label. How would mobile and tablet users, or a keyboard only user access the text?

* https://www.tpgi.com/using-the-html-title-attribute-updated/
* https://inclusive-components.design/tooltips-toggletips/

We're hoping to eventually move away from `title` altogether, https://github.com/carbon-design-system/carbon/issues/9208